xpk.py

```diff
@@ -1180,12 +1230,12 @@ def run_gke_node_pool_create_command(args, system_characteristics) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
-
+  device_type = args.tpu_type if args.tpu_type else args.gpu_type
```

Comment:
Lets had a new variable which is the AcceleratorType. Then we can pass this AcceleratorType to internal functions. This will simplify logic your helper functions. Currently the helper functions all have to check the AcceleratorType, with this, we just check once.

Reply:
yeah you mean to add args.accelerator_type right? so we would expect user to provide both accelrator_type and device_type then?